examples/features/policy-based-routing/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ns-policy-based-routing
+
+resources:
+- client.yaml
```

Comment:
I think we should use the same notation in kustomize files - either relative path or full links.

In my opinion, relative paths are less prone to errors.